src/org/traccar/model/Position.java

```diff
@@ -54,6 +54,7 @@
     public static final String KEY_APPROXIMATE = "approximate";
     public static final String KEY_THROTTLE = "throttle";
     public static final String KEY_MOTION = "motion";
+    public static final String KEY_GUARD = "guard";
```

Comment:
Yes, I think "armed" is a better English word for it. Also, we use it for commands already, so it's also better for consistency.